bfvmm/include/memory_manager/buddy_allocator.h

```diff
@@ -116,40 +109,40 @@ class buddy_allocator
         size_type size;
     };
 
-    auto get_size(node_t *node) const
+    inline auto get_size(node_t *node) const
```

Comment:
Is there a particular reason to use signed types here?

Reply:
Nope. I will fix that in my next PR.